diff --git a/ios/GlassContainer.swift b/ios/GlassContainer.swift
index 44513a7b7a8beb2644300c3622f332943f6a0d8a..ca8708a908753b33b365d8de6f6fb9f3d22dd74d 100644
--- a/ios/GlassContainer.swift
+++ b/ios/GlassContainer.swift
@@ -43,11 +43,11 @@ public final class GlassContainer: ExpoView {
     }
   }
 
-  public override func mountChildComponentView(_ childComponentView: UIView, index: Int) {
+  public func mountChildComponentView(_ childComponentView: UIView, index: Int) {
     containerEffectView.contentView.insertSubview(childComponentView, at: index)
   }
 
-  public override func unmountChildComponentView(_ childComponentView: UIView, index: Int) {
+  public func unmountChildComponentView(_ childComponentView: UIView, index: Int) {
     childComponentView.removeFromSuperview()
   }
 }
diff --git a/ios/GlassView.swift b/ios/GlassView.swift
index 794bdb43028d09e20d8aa20c91c64b5913d6b039..d60edb00cfb1f12c3fe031b2b0be9d569cc4989e 100644
--- a/ios/GlassView.swift
+++ b/ios/GlassView.swift
@@ -271,11 +271,11 @@ public final class GlassView: ExpoView {
       #endif
     }
   }
-  public override func mountChildComponentView(_ childComponentView: UIView, index: Int) {
+  public func mountChildComponentView(_ childComponentView: UIView, index: Int) {
     glassEffectView.contentView.insertSubview(childComponentView, at: index)
   }
 
-  public override func unmountChildComponentView(_ childComponentView: UIView, index: Int) {
+  public func unmountChildComponentView(_ childComponentView: UIView, index: Int) {
     childComponentView.removeFromSuperview()
   }
 }
