diff --git a/node_modules/react-native-date-picker/ios/RNDatePicker.mm b/node_modules/react-native-date-picker/ios/RNDatePicker.mm
index 7ec2dd2..3b9bf76 100644
--- a/node_modules/react-native-date-picker/ios/RNDatePicker.mm
+++ b/node_modules/react-native-date-picker/ios/RNDatePicker.mm
@@ -1,3 +1,5 @@
+#include <string>
+
 #import "RNDatePicker.h"
 
 #ifdef RCT_NEW_ARCH_ENABLED
diff --git a/node_modules/react-native-date-picker/ios/RNDatePickerManager.mm b/node_modules/react-native-date-picker/ios/RNDatePickerManager.mm
index da42873..4564b4d 100644
--- a/node_modules/react-native-date-picker/ios/RNDatePickerManager.mm
+++ b/node_modules/react-native-date-picker/ios/RNDatePickerManager.mm
@@ -1,3 +1,5 @@
+#include <string>
+
 #import <React/RCTLog.h>
 #import <React/RCTUIManager.h>
 #import <React/RCTViewManager.h>
